src/api/middleware/validators/validatorUtils.js

```diff
@@ -50,9 +52,22 @@ const isObjectId = (id) => {
     return true;
 };
 
+/**
+ * Checks if the concurrent offers of a given owner have not exceeded the defined limit
+ * @param {*} offerInstance
+ */
+const concurrentOffersNotExceeded = (offerInstance) => async (owner, publishDate, publishEndDate) => {
```

Comment:
```suggestion
const concurrentOffersNotExceeded = (OfferModel) => async (owner, publishDate, publishEndDate) => {
```

That variable refers to the model, and not to the instances. Hence the uppercase change as well